src/tests/ftest/container/autotest.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from apricot import TestWithServers
+from command_utils_base import CommandFailure
+
+
+class ContainerAutotestTest(TestWithServers):
+    # pylint: disable=too-few-public-methods
+    """Tests container autotest.
+
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize object."""
+        super().__init__(*args, **kwargs)
+        self.daos_cmd = None
```

Comment:
This is not required and can be removed to simplify things.  The `self.daos_cmd` only needs to be a class attribute if it is accessed across multiple test methods.  As there is only one test method using it - `test_container_autotest()` it can be a local method variable.